silkworm/db/kv/txn_num.hpp

```diff
@@ -56,7 +56,8 @@ Task<std::optional<BlockNum>> block_num_from_tx_num(kv::api::Transaction& tx,
 struct TransactionNums {
     TxnId txn_id{0};
     BlockNum block_num{0};
-    TxnId txn_index{0};
+    std::optional<uint32_t> txn_index{0};
```

Comment:
I would use empty initialisation for `std::optional<uint32_t>`

Reply:
ok